cmd/podman/main_local.go

```diff
@@ -120,6 +120,14 @@ func profileOff(cmd *cobra.Command) error {
 }
 
 func setupRootless(cmd *cobra.Command, args []string) error {
+	matches, err := rootless.ConfigurationMatches()
+	if err != nil {
+		return err
+	}
+	if !matches {
+		logrus.Warningf("the user namespace doesn't match the configuration in /etc/subuid or /etc/subgid")
```

Comment:
the *current* user namespace doesn't match the configuration in /etc/subuid or /etc/subgid